tools/mo/openvino/tools/mo/back/offline_transformations.py

```diff
@@ -71,6 +71,23 @@ def read_model(path_to_xml):
 
     func = read_model(input_model + "_tmp.xml")
 
+    if argv.framework == 'tf' and argv.tensorflow_custom_operations_config_update:
+        return func
```

Comment:
Why is this code needed?

Reply:
We should separately handle this case to have a fallback for this.

Reply:
Originally this check was in emit_ir(). If this check was true offline transformations were not applied.
But now apply_offline_transformations() generates ov.Model after reading _tmp ir. So this method should be executed in all cases and return ov.Model. So I moved this check into the method to read the model but not apply offline transformations.

Reply:
Here is original check: https://github.com/openvinotoolkit/openvino/blob/e2080cd8a458fcb650e5a0bc8477ec1416dc525d/tools/mo/openvino/tools/mo/main.py#L438

Reply:
I am not sure why this check is moved here if we have it in upper level `emit_ir` for legacy frontends. Now you move this to common place for both frontend. Not sure what is it needed. Please clarify

Reply:
Moved this check to emit_ir as discussed offline, and made apply_offline_transformations() to accept openvino.runtime.Model.